src/KOKKOS/pair_mliap_kokkos.cpp

```diff
@@ -233,13 +233,15 @@ void PairMLIAPKokkos<DeviceType>::coeff(int narg, char **arg) {
   // map[i] = which element the Ith atom type is, -1 if not mapped
   // map[0] is not used
 
+  //printf(">>> ntypes: %d\n", atom->ntypes);
```

Comment:
This needs to be removed or guarded so only one rank prints it out and printed to both screen and logfile as done in other places.

Reply:
Got it, I removed it!